emitter.ts

```diff
@@ -87,7 +87,11 @@ export async function emitter(
       emitDestinations.length > 0
         ? `Loaded emit destinations, ${
           emitDestinations.map((dest) =>
-            `address: ${dest.sourceAddress}  event signature hash: ${dest.abiHash}  topic filters: [1] ${dest.topic1} [2] ${dest.topic2} [3] ${dest.topic3}  destination: ${dest.webhookUrl}`
+            `address: ${toHex(dest.sourceAddress)}  event signature hash: ${
+              toHex(dest.abiHash)
+            }  topic filters: [1] ${toHex(dest.topic1)} [2] ${
+              toHex(dest.topic2)
+            } [3] ${toHex(dest.topic3)}  destination: ${dest.webhookUrl}`
```

Comment:
Seems nullish check for topics is needed before invoking toHex

(+ I would prefer the way like L300 below)

Reply:
Amended!